HTML:CSS learning project/index.html

```diff
@@ -23,69 +25,43 @@
           <li class="menu-item"><a href="#">Contact Us</a></li>
         </ul>
       </nav>
-      <hr style="width: 100%"/>
+      <hr style="width: 100%" />
     </div>
   </header>
-  <main> 
+  <main>
     <section class="hero">
-          <h1 class="title">ROAD TO WEB DEVELOPER</h1> 
-      </section> 
-      <div class="button">
-        <a href="https://www.youtube.com/watch?v=UTfRLdY_8AU" class="title-button">First button</a>
-        <a href="https://www.youtube.com/watch?v=sfYUdITM4Qk" class="title-button">Second button:(</a>
-      </div>
+      <h1 class="title">ROAD TO WEB DEVELOPER</h1>
+    </section>
+    <div class="button">
```

Comment:
this class is not semantically correct. This div isn't button, it is a wrapper for buttons. Try to change class name.